templates/data/opensearch/support.html

```diff
@@ -194,6 +194,49 @@ <h3 class="p-heading--5">Fast response time</h3>
     </div>
   </section>
 
+  <section class="p-section">
+    <div class="row p-section--shallow">
+      <hr class="p-rule" />
+      <div class="col-6">
+        <h2>Multiple packaging formats</h2>
+      </div>
+      <div class="col-6">
+        <p>
+          We provide maintained and supported OpenSearch packages in both snap and container formats:
+        </p>
+      </div>
+    </div>
```

Comment:
I think it's best to use 50-50 split here. This makes the row responsive without needing to add additional col classes. Also, please wrap the row in the `p-section--shallow` instead, it's not recommended to mix grid and component classes like this

```suggestion
  <div class=p-section--shallow">
    <div class="row--50-50">
        <hr class="p-rule" />
        <div class="col">
          <h2>Multiple packaging formats</h2>
        </div>
        <div class="col">
          <p>
            We provide maintained and supported OpenSearch packages in both snap and container formats:
          </p>
        </div>
      </div>
  </div>
    
```